src/components/EFTs.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import ShowButton from './ShowButton';
+import { categories } from './data';
+import Loading from './Loading';
+import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css';
+import '../styles/EFTs.css';
+
+function EFTs() {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```